index.html

```diff
@@ -305,6 +305,53 @@ <h3 class="title-about">
           </ul>
         </section>
       </section>
+
+      <!-- contact section -->
+      <footer class="footer">
+        <h2 class="contact-title title">Contact me</h2>
+        <article class="contact-info">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </article>
+        <form
+          action="https://formspree.io/f/xknydra"
+          method="post"
+          class="input form"
+        >
+          <input
+            type="text"
+            class="text"
+            placeholder="Yeremias"
+            maxlength="28"
```

Comment:
- [ ] As stated in the project requirement Kindly Make sure that: Name field accepts up to 30 chars.So Kindly update the `maxlength="30"`